cockatrice/src/settingscache.h

```diff
@@ -47,6 +47,8 @@ class SettingsCache : public QObject {
     QByteArray mainWindowGeometry;
     QString lang;
     QString deckPath, replaysPath, picsPath, cardDatabasePath, tokenDatabasePath, themeName;
+    QString handBgPath, stackBgPath, tableBgPath, playerBgPath, cardBackPicturePath;
```

Comment:
I noticed that this line is no longer used and was possibly removed in the recent theme manager PR commit. @ctrlaltca can you confirm that I can remove this line?  I don't want to add it back if none of the variables are no longer needed.